dandiapi/api/views/version.py

```diff
@@ -79,7 +79,9 @@ def retrieve(self, request, **kwargs):
     def info(self, request, **kwargs):
         """Django serialization of a version."""
         version = self.get_object()
-        serializer = VersionDetailSerializer(instance=version)
+        serializer = VersionDetailSerializer(
+            instance=version, context=self.get_serializer_context()
+        )
```

Comment:
@mvandenburgh @waxlamp I don't love this solution, but it works. I was curious how the other serializers work without this (e.g. the Dandiset retrieve endpoint), and it turns out the DRFViewSet class we inherit from (specifically `ReadOnlyModelViewSet` -> `GenericViewSet` -> `generics.GenericAPIView`) automatically populate this context when the `get_serializer` method is called. This means that on all untouched methods we inherit and use, it's populated, while on any methods we've overwritten, it's missing (unless we explicitly specify it, like it done here).

To me this suggests that we should do an overarching refactor of our view and serialization code, since I think we have a lot of technical debt there. But for now, this is what seems to be the easiest change that's in scope.